iocore/net/SSLUtils.cc

```diff
@@ -2475,3 +2329,150 @@ SSLConnect(SSL *ssl)
 
   return ssl_error;
 }
+
+bool
+SSLMultiCertConfigLoader::load_certs(SSL_CTX *ctx, std::vector<X509 *> &certList, const SSLConfigParams *params,
+                                     const SSLMultiCertConfigParams *sslMultCertSettings)
+{
+  SimpleTokenizer cert_tok((const char *)sslMultCertSettings->cert, SSL_CERT_SEPARATE_DELIM);
```

Comment:
The next tidy up could be to replace the Tokenizer with a stringview/textview based parser.  Eliminate inserting nulls into the string or copying out.